### PR TITLE
edited general content, options and added video/image formats

### PR DESCRIPTION
**Pull request by bernie-xailient**

### CODE REVIEW

File: docs/container.md [hunk 1]
```diff
@@ -62,23 +62,26 @@ run through the Xailient SDK.
...
+    `$ docker run -e SDK_LINK="https://ReallyLongUrl/AAaDDY2MzE1ODQ0NzQx" -v ~/Desktop/input:/input -v ~/Workspace/output:/output xailient/model-inference:1.0.1`
 
-All predicted images and videos will appear in the `~/Workspace/output` of the host OS.
+In this example, all predicted images and videos will appear in the `~/Workspace/output` of the host OS.
 
 <br>
 ## Default Behaviour
 
-By default the Xailient SDK container will output images for all given input images, and output videos for all given input videos.
+By default the Xailient SDK container will output images for all given input images, and output videos and their frames for all given input videos.
```
Comment (bernie-xailient) [inline docs/container.md:81]: My original section of example images and videos disappeared so didn't think it was worth actually giving the user example images anymore.

File: docs/container.md [hunk 2]
```diff
@@ -62,23 +62,26 @@ run through the Xailient SDK.
...
+
+* Supported Image formats: jpeg/jpg, png
+* Supported Video formats: mp4, avi, flv, mkv, mov, webm, wmv
```
Comment (bernie-xailient) [inline docs/container.md:84]: New video formats supported

File: docs/container.md [hunk 3]
```diff
@@ -99,28 +102,23 @@ To disable this set the environment variable IMAGES to False.
 Controls whether videos with predicted bounding boxes are created in the output directory for input videos.
 See the `FPS` and `EVERY` option for even more control over the output video that gets created.
 
-If `REBUILD` is False then one output image is created for each frame of the video. The output image names will be the same
-name as the video file, with a `_1`, `_2`, etc appended. 
-So an input video named `FrontDoor-Oct-14-2020.mp4` will cause output images named `FrontDoor-Oct-14-2020_1.jpg`,
-`FrontDoor-Oct-14-2020_2.jpg`, etc to be created in the output directory.
```
Comment (bernie-xailient) [inline docs/container.md:105]: When REBUILD is false, videos are simply not rebuilt. The inference on frames is separate to this and is controlled by the IMAGES option. This allows the user to have both videos and individual frames without having to rerun the container.

File: docs/container.md (hunk 1 above)
Comment (jim-meyer) [reply docs/container.md:81]: Hmm, did the default for videos really change to output both a video and each frame like this edit suggests?

File: docs/container.md (hunk 3 above)
Comment (jim-meyer) [reply docs/container.md:105]: The desired default behavior here is that input images -> output images and input videos -> output videos.  We don't want videos producing frames and video both by default.

File: docs/container.md [hunk 4]
```diff
@@ -99,28 +102,23 @@ To disable this set the environment variable IMAGES to False.
...
-### FPS
+### FPS (defaults to the original video fps)
 Reconstructed videos will have this many frames per second.
 
 **Example:**
 `$ docker run -e FPS=30 -v <FULL_PATH>:/input -v <FULL_PATH>:/output xailient/model-inference:1.0.1`
 
 ### THRESHOLD (default=0.5)
-Play around with different confidence thresholds by passing in the environment variable THRESHOLD when running the Container instance.
+Play around with different confidence thresholds by passing in the environment variable THRESHOLD when running the Container instance. Accepts floating values from 0 to 1.
```
Comment (jim-meyer) [inline docs/container.md:121]: Good clarification

**Review by jim-meyer — APPROVED** [review]

**Pull request merged**